src/05-objects.js

```diff
@@ -118,7 +137,12 @@ function addKeyToObject(object, key) {}
  *  addKeyValuePairToObject({ up: true }, "left", false);
  *  //> { up: true, left: false }
  */
-function addKeyValuePairToObject(object, key, val) {}
+function addKeyValuePairToObject(object, key, val) {
+  // let pushObj = {[key]:[val]}
```

Comment:
Lets get into the habit of removing commented out code.  Comments are ok but unused code is not